configuration/serialization-serde.md

```diff
@@ -73,6 +73,7 @@ kafka:
       serde:
         - name: ProtobufFile
           properties:
+            className: io.kafbat.ui.serdes.builtin.ProtobufFileSerde
```

Comment:
This shouldn't be needed as proto is a built-in serde, given the name stays as is, no classname should be specified (`io/kafbat/ui/serdes/SerdesInitializer.java:44`)